en/08_Changelogs/6.1.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 6.0.0 (unreleased)
+---
+
+# 6.0.0 (unreleased)
+
+## Overview
+
+- [Features and enhancements](#features-and-enhancements)
+  - [Non-blocking file-based sessions](#non-blocking-sessions)
+  - [Other new features](#other-new-features)
+- [API changes](#api-changes)
+- [Bug fixes](#bug-fixes)
+
+## Features and enhancements
+
+## Non-blocking file-based sessions {#non-blocking-sessions}
+
+The default file-based session handler for PHP holds a lock on the session file while the session is open. In practice this means that multiple concurrent requests from the same user have to wait for one another to finish processing after a session has been started. This includes AJAX requests.
```

Comment:
```suggestion
The default file-based session handler for PHP holds a lock on the session file while the session is open. This means that multiple concurrent requests from the same user have to wait for one another to finish processing after a session has been started. This includes XHR requests.
```

Reply:
XHR vs AJAX discussion can be held in https://github.com/silverstripe/developer-docs/pull/733#discussion_r2074381494

Reply:
Made the other change.